file_paths.md

```diff
@@ -161,6 +165,11 @@ create a `puppet` user or group.
     Path                                      Setting                        3.x
     C:\ProgramData                                                           n/a
 
+    C:\ProgramData\PuppetLabs/client-tools    # default client tool settings
```

Comment:
Ah, i'll fix the copypasta error. I'm not sure what else would end up there, but presuming there is another tool every release, and we don't do the work to consolidate all client configs into one file (which would be the goal, but here we are), there would be the 1:1 correspondence between tool and config file.